src/deprecates.jl

```diff
@@ -48,3 +48,15 @@ findat(a::AbstractArray, b::AbstractArray) = findat!(Array{Int}(size(b)), a, b)
 @deprecate df_residual(obj::StatisticalModel) dof_residual(obj)
 
 @deprecate_binding WeightVec Weights
+
+using Base.Random: RangeGenerator
+@deprecate_binding RandIntSampler RangeGenerator
+@deprecate RangeGenerator(K::Int) RangeGenerator(1:K)
```

Comment:
That's a tricky situation. I first tried to do `@deprecate RandIntSampler(K::Int) RangeGenerator(1:K)`, but that triggers a deprecation warning when loading StatsBase. So that's the only solution I could find. That's a kind of type piracy, but hopefully we won't have to keep this for too long.